keras_cv/bounding_box/iou.py

```diff
@@ -172,3 +174,71 @@ def compute_iou(
     )
     iou_lookup_table = tf.where(background_mask, mask_val_t, res)
     return iou_lookup_table
+
+
+def compute_ciou(box1, box2, bounding_box_format, eps=1e-7):
```

Comment:
Since this is now defined here, let's drop it from `ciou_loss.py` and just import it from here instead

Reply:
Should we leave 'ciou_loss.py' as a loss in the losses folder given that developers can use it for their own purposes/tasks? wdyt @ianstenbit

Reply:
Yes! Sorry I was ambiguous with my comment -- we should still offer `CIoULoss`, we should just import the `compute_ciou` method from here instead of having a second copy of it in `ciou_loss.py`